src/main/kotlin/de/byteleaf/companyon/common/graphql/GraphQLOffsetDateTime.kt

```
@@ -0,0 +1,50 @@
+package de.byteleaf.companyon.common.graphql
+
+import graphql.language.StringValue
+import graphql.schema.*
+import org.springframework.stereotype.Component
+import java.time.OffsetDateTime
+import java.time.format.DateTimeFormatter
+import java.time.format.DateTimeParseException
+
+
+@Component
+class GraphQLOffsetDateTime @JvmOverloads constructor(name: String? = DEFAULT_NAME) :
+    GraphQLScalarType(name, "OffsetDateTime", object : Coercing<OffsetDateTime, String> {
+        private fun convertImpl(input: Any): OffsetDateTime? {
+            if (input is String) {
+                try {
+                    return OffsetDateTime.parse(input, DateTimeFormatter.ISO_OFFSET_DATE_TIME)
+                } catch (ignored: DateTimeParseException) {
+                    // nothing to-do

```

should throw an exception